### ISSUE TITLE
Hyper-V: Show preview VNC by default

### ISSUE THREAD

**Issue by mcandre** [hashicorp/packer#5603]
FOR FEATURES:

Like VirtualBox, VMware, qemu, and other providers, it would be helpful for debugging builds if the Hyper-V provider would also show a VNC preview, using the Hyper-V Manager GUI.

Note that Hyper-V Manager shows a loud, awkward complain prompt when VM's packer deletes a VM, maybe there is a way to disable this warning?

**Comment by kikitux** [comment]
Hello

I am confused on what you say about `by default`

do you have VNC today in hyper-v?

PR welcome.

**Comment by mcandre** [comment]
As a user, I want packer to show a graphical preview window during Hyper-V builds, just like packer does for VirtualBox and VMware builds, so that I can more easily see what the guest is doing during the critical `boot_command` phase and debug it.

**Comment by zoredache** [comment]
I don't believe the OP is actually looking for VNC.  Instead they are looking to have the vmconnect launched.  So if the local machine was named `hypervhost`, and the client vm was named `packertemp` you might run a command like this.

    "C:\WINDOWS\system32\VmConnect.exe" "hypervhost" "packertemp"
